lib/studies/reply/search_page.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flutter/material.dart';
+
+class SearchPage extends StatelessWidget {
+  const SearchPage();
+
+  @override
+  Widget build(BuildContext context) {
+    final splashRadius = 24.0;
+
+    return Scaffold(
+      body: SafeArea(
+        child: Material(
+          color: Theme.of(context).colorScheme.surface,
+          child: Column(
+            children: [
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Row(
+                  mainAxisSize: MainAxisSize.max,
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    IconButton(
+                      icon: const Icon(Icons.arrow_back),
+                      onPressed: () => Navigator.of(context).pop(),
+                      splashRadius: splashRadius,
+                    ),
+                    const Expanded(
+                      child: TextField(
+                        decoration: InputDecoration.collapsed(
+                          hintText: 'Search email',
+                        ),
+                      ),
+                    ),
+                    IconButton(
+                      icon: const Icon(Icons.mic),
+                      onPressed: () {},
+                      splashRadius: splashRadius,
+                    )
+                  ],
+                ),
+              ),
+              const Divider(thickness: 1.1),
```

Comment:
1.1 seems like a weird number to use here, what's the reason?

Reply:
I had an issue before hand where text and sometimes lines on the web were getting cut off if the height of the text wasn't changed to something higher than 1.0. It may have just been that special case though because changing this to 1.0 works.